### ISSUE TITLE
How to assign values ​​to an element of a vector?

### ISSUE THREAD

**Issue by AlainBarrios** [martinlaxenaire/curtainsjs#46]
I was observing that in new versions of the library I assign values ​​to the items but it doesn't detect. 
Another problem is that now i can't use gsap to change vector values

this is a example not works
```js
this.bufferTrail.uniforms.mousepos.value[0] = this.mouse.x
this.bufferTrail.uniforms.mousepos.value[1] = this.mouse.y
```

this works
```js
this.bufferTrail.uniforms.mousepos.value = [this.mouse.x, this.mouse.y]
```

**Comment by martinlaxenaire** [comment]
Hey @AlainBarrios, 

you're right, it seems we can't update array uniforms values independently.
I've managed to fix this bug. I'll close this issue as soon as I find some time to commit the update.

Thanks!

Cheers,

**Comment by martinlaxenaire** [comment]
Fixed with version 6.2.2
